safe/utilities/rounding.py

```diff
@@ -0,0 +1,95 @@
+# coding=utf-8
+
+"""Rounding and number formatting.
+
+Taken from InaSAFE version 4.
+"""
+
+from safe.utilities.i18n import locale
+
+__copyright__ = "Copyright 2016, The InaSAFE Project"
+__license__ = "GPL version 3"
+__email__ = "info@inasafe.org"
+__revision__ = '$Format:%H$'
+
+
+def thousand_separator():
+    """Return thousand separator according to the locale.
+
+    :return: The thousand separator.
+    :rtype: basestring
+    """
+    lang = locale()
+
+    if lang in ['id']:
+        return '.'
+
+    elif lang in ['fr']:
+        return ' '
+
+    else:
+        return ','
+
+
+def add_separators(x):
+    """Format integer with separator between thousands.
+
+    :param x: A number to be formatted in a locale friendly way.
+    :type x: int
+
+    :returns: A locale friendly formatted string e.g. 1,000,0000.00
+        representing the original x. If a ValueError exception occurs,
+        x is simply returned.
+    :rtype: basestring
+
+    From http://
+    stackoverflow.com/questions/5513615/add-thousands-separators-to-a-number
+
+    Instead use this:
+    http://docs.python.org/library/string.html#formatspec
+    """
+    try:
+        s = '{0:,}'.format(x)
+        # s = '{0:n}'.format(x)  # n means locale aware (read up on this)
+    # see issue #526
+    except ValueError:
+        return x
+
+    # Quick solution for the moment
+    if locale() in ['id', 'fr']:
+        # Replace commas with the correct thousand separator.
+        s = s.replace(',', thousand_separator())
+    return s
+
+
+def fatalities_range(number):
+    """A helper to return fatalities as a range of number.
+
+    See https://github.com/inasafe/inasafe/issues/3666#issuecomment-283565297
+
+    :param number: The exact number. Will be converted as a range.
+    :type number: int, float
+
+    :return: The range of the number.
+    :rtype: str
+    """
+    range_format = '{min_range} - {max_range}'
+    more_than_format = '> {min_range}'
+    ranges = [
+        [0, 100],
+        [100, 1000],
+        [1000, 10000],
+        [10000, 100000],
+        [100000, float('inf')]
+    ]
```

Comment:
This is a different range distribution.
So I guess it was supposedly per 10 instead of per 100?

Reply:
Yes, I think so. I am not the best person with this range, but since it's in develop, I expect @Charlotte-Morgan wants to have it also in realtime.

Reply:
Hi @lucernae - is there some where you can share what the realtime range was before this ticket?
I guess we do not have many events that have used anything except the lower ranges before.

Reply:
Hi @Charlotte-Morgan 
I checked the history of the code, it's back to 5 years ago (from @timlinux ) in this commit: https://github.com/ismailsunni/inasafe-realtime/commit/4973a1d8ca74a1f72ccf32df024778b55ff4536a

If I checked the code's comment, it's the same the expected:
> # put the estimate into neat ranges 0-100, 100-1000, 1000-10000. etc

But, checking the code, there is something wrong, in this line:
```
    myUpperLimit = math.pow(myUpperLimit, 2)
```
That's why we got 10.000, not 1.000 in the range (it was 100 ^ 2, it should be 100 * 10).

So, I think in this PR we fix a bug that only happen if the fatality > 100.